apps/common/views/partials/back.html

```diff
@@ -0,0 +1,5 @@
+{{$back}}
+  {{#backLink}}
+    <a href="{{#url}}{{backLink}}{{/url}}">Back</a>
```

Comment:
I need to create a new common translation file as the current ones (buttons, errors and errorlist) do match this type of translation. What do you suggest? partials.json? pages.json?